src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -11,6 +11,8 @@
 import java.util.Date;
 import java.util.List;
 import java.util.logging.Logger;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
```

Comment:
Ahhh, good catch, I forgot to remove after changing the replacement! I'll fix that soon :)